install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -49,9 +49,13 @@ data:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
           capabilities:
+            drop:
+            - "all"
             add:
-            - NET_ADMIN
-          privileged: true
+            - "NET_ADMIN"
+            - "NET_RAW"
+          privileged: false
```

Comment:
Priviledged appear to be required by openshift ( and possibly other ), Andra is trying to add an install option so it is enabled only on those environments - but we can't turn it off and break platforms without an option.

Reply:
Let's keep privilege out of this PR, it is addressed in #7890.

I like the changes for allowPrivilegeEscalation: false, because this defaults to true when priviledge=true, so we need to explicitly set it to false.